client/src/utils/filtering.ts

```diff
@@ -61,6 +69,14 @@ export function expandNameTag(value: string | object): string {
     return toLower(value);
 }
 
+/** Converts string alias to string operator, e.g.: 'gt' to '>'
+ * @param {string} alias
+ * @returns {string} Arithmetic operator, e.g.: '>'
```

Comment:
Remove duplicate type declarations.

```suggestion
 * @param alias
 * @returns Arithmetic operator, e.g.: '>'
```